src/main/java/uk/gov/hmcts/reform/professionalapi/controller/advice/ExceptionMapper.java

```diff
@@ -83,6 +85,13 @@
         return errorDetailsResponseEntity(ex, CONFLICT, CONFLICT_EXCEPTION.getErrorMessage());
     }
 
+    @ExceptionHandler(HttpClientErrorException.class)
+    public ResponseEntity<Object> handleHttpClientErrorException(
+            HttpClientErrorException ex) {
+        return errorDetailsResponseEntity(ex, CONFLICT, DUPLICATE_USER.getErrorMessage());
```

Comment:
since this is specifically used for throwing duplicate user exception..can we not name this as handleDuplicateUserException instead of handleHttpClientErrorException

Reply:
ok , can be done.